types/material-ui/material-ui-tests.tsx

```diff
@@ -2764,6 +2764,17 @@ class DropDownMenuOpenImmediateExample extends React.Component<{}, {value?: numb
   }
 }
 
+const DropDownMenuAnchorExample: React.SFC<{}> = () => (
```

Comment:
I've only tested the `origin` props since that's the functionality I need. The other props have been copied from the documentation for v0.17.2 but haven't been explicitly tested